libs/SmartSync/SmartSync/Classes/Util/SFSmartSyncNetworkUtils.m

```diff
@@ -32,8 +33,15 @@
 @implementation SFSmartSyncNetworkUtils
 
 + (void)sendRequestWithSmartSyncUserAgent:(SFRestRequest *)request failBlock:(SFRestFailBlock)failBlock completeBlock:(SFRestResponseBlock)completeBlock {
-     [request setHeaderValue:[SFRestAPI userAgentString:kSmartSync] forHeaderName:kUserAgent];
-    [[SFRestAPI sharedInstance] sendRESTRequest:request failBlock:failBlock completeBlock:completeBlock];
+    LogSyncDebug(@"sendRequestWithSmartSyncUserAgent:request:%@", request);
+    [request setHeaderValue:[SFRestAPI userAgentString:kSmartSync] forHeaderName:kUserAgent];
+    [[SFRestAPI sharedInstance] sendRESTRequest:request failBlock:^(NSError *e) {
+        LogSyncError(@"sendRequestWithSmartSyncUserAgent:error:%ld:%@", (long) e.code, e.domain);
+        failBlock(e);
+    }                             completeBlock:^(id response) {
+        LogSyncDebug(@"sendRequestWithSmartSyncUserAgent:response:%@", response);
```

Comment:
When log level is debug - we get to see all the network traffic as well as operations against the local store - it makes following what's going on a lot easier.